src/main/java/com/parkhomovsky/bookstore/dto/order/OrderPlaceRequestDto.java

```diff
@@ -0,0 +1,10 @@
+package com.parkhomovsky.bookstore.dto.order;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+
+@Data
+public class OrderPlaceRequestDto {
+    @NotNull
+    private String shippingAddress;
```

Comment:
can the string be empty? Also, what if user will input string which is longer than maximum string length in DB?